app/controllers/RegisterTrustController.scala

```diff
@@ -73,7 +73,7 @@ class RegisterTrustController @Inject()(
         case Right(registration) =>
           register(registration)
         case Left(validationErrors) =>
-          logger.error(s"[Session ID: ${request.sessionId}] problem validating submission: $validationErrors")
+          logger.warn(s"[RegisterTrustController][validateRegistration][Session ID: ${request.sessionId}] problem validating submission: $validationErrors")
```

Comment:
Any context as to why this went down to warn?

Reply:
I will need to retrace, but this should be logged elsewhere in the service as an error, made a warning so we don't end up with multiple errors for the same failure

Reply:
just confirmed it, this is raised as an error within the function call on line 72